kubejs/server_scripts/base/unification/unify_stoneworks.js

```diff
@@ -0,0 +1,92 @@
+//priority: 900
+ServerEvents.recipes(event => {
+    generatableCobblestone.forEach((material) => {
+        var type = 'cobble';
+        //console.log(`Recipe for Material: ${material}, Type: ${type}`);
+        //pedestals_stoneworks(event, material, type);
+        //industrialforegoing_stoneworks(event, material, type);
+        //thermal_stoneworks(event, material);
+    });
+
+    generatableStone.forEach((material) => {
+        var type = 'stone';
+        //console.log(`Recipe for Material: ${material}, Type: ${type}`);
+        //pedestals_stoneworks(event, material, type);
+        //industrialforegoing_stoneworks(event, material, type);
+        //thermal_stoneworks(event, material);
+    });
+});
+
+//stoneworks functions
+function pedestals_stoneworks(event, material, type) {
+    if (excludeDuplicatesCobbleGen.includes(material)) {
+        return;
+    }
+
+    var recipeType = 'pedestals:pedestal_cobblegen';
+
+    if (type == 'stone') {
+        recipeType = 'pedestals:pedestal_cobblegensilk';
+    }
+    //console.log(`Pedestals Recipe for Material: ${material}, Type: ${type}`);
+    fallback_id(
```

Comment:
Let's remove fallback_id and do things with ids this time around :D

Reply:
✅